ydb/core/protos/flat_scheme_op.proto

```diff
@@ -1806,6 +1806,7 @@ message TExternalTableDescription {
     optional string Location = 6;
     repeated TColumnDescription Columns = 7;
     optional bytes Content = 8;
+    optional bool ReplaceIfExists = 9;
```

Comment:
This field has no sense for describe operation. It's worth to write comment about it.